Assets/Scripts/Calendar UI.cs

```diff
@@ -80,46 +102,114 @@ private void HandleBackwardClick()
         SetNumberOfBlanks();
         PopulateCalendarGrid();
     }
+    private void HandleCellImageSelect(Cell cell)
+    {
+        foreach (DayCell c in populationList)
+        {
+            if (c != cell)
+            {
+                c.DeSelect();
+            }
+        }
+    }
+    private void HandleSelectCell(Cell cell)
+    {
+        if (pop == null)
+        {
+
+            selectedCell = cell as DayCell;
+
+            pop = Instantiate(popUpPrefab, canvas);
+            pop.MakeButton("Sticker", HandleAddStickerPressed, false);
+            pop.MakeButton("Notes", HandleAddDescriptionPressed, false);
+            pop.SetPosition(cell.transform.position);
+
+
+        }
+    }
+    private void HandleAddDescriptionPressed()
+    {
+        EventManager.Instance.TriggerAddDescriptionPressed();
+        //Make function to trigger description scene;
+    }
+
 
-    public void PopulateCalendarGrid()
+    private void HandleAddStickerPressed()
     {
 
+        pop.DestroyButonCells();
+        pop.MakeButton("Poop", HandleAddPoopPressed, selectedCell.DData.isPoopImageActive);
+        pop.MakeButton("Pill", HandleAddMedicinePressed, selectedCell.DData.isMedicineImageActive);
+
+
+    }
+
+    private void HandleDESelectAllCells()
```

Comment:
This function name confuses me as this isnt deselecting all cells, just the selectedCell

Reply:
Resolved